files/pt-br/web/api/audio_output_devices_api/index.md

```diff
@@ -0,0 +1,107 @@
+Esta API permite que as aplicações ofereçam essa mesma funcionalidade a partir de uma página web.
+
+Mesmo se permitido por uma política de permissões, o acesso a um dispositivo de saída de áudio específico ainda requer permissão explícita do usuário, pois o usuário pode estar em um local onde a reprodução de áudio através de alguns dispositivos de saída não seja apropriada.
+
+A API fornece o método [`MediaDevices.selectAudioOutput()`](/pt-BR/docs/Web/API/MediaDevices/selectAudioOutput) que permite aos usuários selecionar sua saída de áudio desejada dentre aquelas permitidas pela diretiva [`speaker-selection`](/pt-BR/docs/Web/HTTP/Headers/Permissions-Policy/speaker-selection) do cabeçalho HTTP [`Permissions-Policy`](/pt-BR/docs/Web/HTTP/Headers/Permissions-Policy) do documento.
+O dispositivo selecionado recebe, então, permissão do usuário, permitindo que ele seja enumerado com [`MediaDevices.enumerateDevices()`](/pt-BR/docs/Web/API/MediaDevices/enumerateDevices) e definido como dispositivo de saída de áudio usando [`HTMLMediaElement.setSinkId()`](/pt-BR/docs/Web/API/HTMLMediaElement/setSinkId).
+
+Dispositivos de áudio podem se conectar e desconectar arbitrariamente. Aplicações que desejam reagir a esse tipo de mudança podem ouvir o evento [`devicechange` event](/pt-BR/docs/Web/API/MediaDevices/devicechange_event) e usar [`enumerateDevices()`](/pt-BR/docs/Web/API/MediaDevices/enumerateDevices) para determinar se `sinkId` está presente nos dispositivos retornados.
+Isso pode iniciar, por exemplo, a pausa ou retomada da reprodução.
+
+## Interfaces
+
+### Extensões para interfaces
+
+A API de Dispositivos de Saída de Áudio estende as seguintes APIs, adicionando as funcionalidades listadas:
+
+#### MediaDevices
+
+- [`MediaDevices.selectAudioOutput()`](/pt-BR/docs/Web/API/MediaDevices/selectAudioOutput)
+  - : Este método solicita ao usuário a seleção de um dispositivo de saída de áudio específico, como um alto-falante ou fone de ouvido.
+    Selecionar um dispositivo concede permissão do usuário para usá-lo e retorna informações sobre o dispositivo, incluindo seu ID.
+
+#### HTMLMediaElement
+
+- [`HTMLMediaElement.setSinkId()`](/pt-BR/docs/Web/API/HTMLMediaElement/setSinkId)
+  - : Este método define o ID do dispositivo de áudio a ser usado para saída, que será usado se permitido.
+- [`HTMLMediaElement.sinkId`](/pt-BR/docs/Web/API/HTMLMediaElement/sinkId)
+  - : Esta propriedade retorna o ID único do dispositivo de áudio em uso, ou uma string vazia se o dispositivo padrão do agente do usuário estiver sendo usado.
+
+## Requisitos de segurança
+
+O acesso à API está sujeito às seguintes restrições:
+
+- Todos os métodos e propriedades só podem ser chamados em um [contexto seguro](/pt-BR/docs/Web/Security/Secure_Contexts).
+
+- [`MediaDevices.selectAudioOutput()`](/pt-BR/docs/Web/API/MediaDevices/selectAudioOutput) concede permissão do usuário para um dispositivo selecionado ser usado como o dispositivo de saída de áudio:
```

Comment:
hi @josielrocha 👋🏻 

are [these flaws](https://github.com/mdn/translated-content/pull/16937#issuecomment-1809993661) due to how I added these URLs? I imagine that since there's a fallback to the `en-us` there's no need to mix URLs from a different language here.

Lmk and I can change it.

should I have done like this instead?

```suggestion
- [`MediaDevices.selectAudioOutput()`](/en-US/docs/Web/API/MediaDevices/selectAudioOutput) concede permissão do usuário para um dispositivo selecionado ser usado como o dispositivo de saída de áudio:
```

Reply:
Valeu @joaodinizgit! ótimo então, vou deixar como está.